src/test/java/racingcar/RacingCarTest.java

```diff
@@ -0,0 +1,87 @@
+package racingcar;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import racingcar.model.Car;
+import racingcar.module.Game;
+import racingcar.module.GameManager;
+import racingcar.util.RandomUtil;
+import racingcar.util.ValidationUtil;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RacingCarTest {
+
+    private ValidationUtil ValidationUtil;
+
+    @DisplayName("입력 값이 숫자가 아닌 값 일때 false 반환")
+    @ParameterizedTest
+    @CsvSource(value = {"1:true" , "d:false" , "1:true"}, delimiter = ':')
+    void inputNumber(String input, boolean result) {
+        //GIVE
+
+        //WHEN
+
+        //THEN
+        assertThat(ValidationUtil.isNumber(input)).isEqualTo(result);
+    }
+
+    @DisplayName("게임 횟수 만큼 Car 객체 생성 크기 비교")
+    @ParameterizedTest
+    @CsvSource(value = {"1:1", "2:2", "3:3"}, delimiter = ':')
+    void carInstanceLength(int input, int result) {
+        //GIVE
+        GameManager gameManager = new GameManager();
+        //WHEN
+
+        //THEN
+        assertThat(gameManager.initCarGame(input)).size().isEqualTo(result);
```

Comment:
`assertj 검증문`을 활용해볼 수 있겠네요 😄 

```suggestion
        assertThat(gameManager.initCarGame(input)).hasSize(result);
```